components/brave_rewards/browser/rewards_service_impl.cc

```diff
@@ -2814,6 +2814,10 @@ bool SaveNormalizedPublisherListOnFileTaskRunner(
     return false;
   }
 
+  if(list.size() == 0) {
```

Comment:
For future reference, this will probably fail our cpp linter. So it should be `if (list.size() == 0)`

Reply:
woops, updated.